app/models/course/assessment/answer/scribing.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+class Course::Assessment::Answer::Scribing < ActiveRecord::Base
+  acts_as :answer, class_name: Course::Assessment::Answer.name
+
+  def to_partial_path
+    'course/assessment/answer/scribing/scribing'
+  end
+
+  MAX_ATTEMPTING_TIMES = 1000
```

Comment:
@weiqingtoh Remove the max attempting times but still keep the attempt limit right? Or do I remove both?